src/styles/App.css

```diff
@@ -0,0 +1,117 @@
+.weather-app {
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: center;
+  min-height: 100vh;
+  padding: 2rem;
+  box-sizing: border-box;
+  color: azure;
+}
+
+.forecast {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: flex-start;
+  margin-top: 2rem;
+  width: 100%;
+  box-sizing: border-box;
+}
+
+.forecast-container {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-items: center;
+}
+
+.forecast-panels {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100%;
+  width: 100%;
+  background: rgba(255, 255, 255, 0.2);
+  border-radius: 16px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
+  border: 1px solid rgba(255, 255, 255, 0.3);
+}
+
+.left-panel,
+.right-panel {
+  width: 50%;
+  height: 100%;
+  padding: 1rem;
+  box-sizing: border-box;
+}
+
+.forecast-summary {
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+  background: rgb(71, 88, 129);
+  background: #475881;
+
+  border-radius: 0.5rem;
+  box-shadow: 0 2px 6px rgba(0, 0, 0, 0.1);
+  padding: 1rem;
+  margin: 1rem;
+  width: calc(33.33% - 2rem);
+  box-sizing: border-box;
+}
+
+.glass-panel {
+  background: rgba(255, 255, 255, 0.2);
+  border-radius: 16px;
+  box-shadow: 0 4px 30px rgba(0, 0, 0, 0.1);
+  backdrop-filter: blur(5px);
+  -webkit-backdrop-filter: blur(5px);
+  border: 1px solid rgba(255, 255, 255, 0.3);
+}
+
+.forecast-summary.selected {
+  width: 100%;
+  justify-content: center;
+  align-items: center;
+  margin: 0%;
+  background: rgb(71, 88, 129);
+}
+
+button {
+  padding: 10px 20px;
+  background-color: white;
+  color: #475881;
+  border: none;
+  border-radius: 5px;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+button:hover {
+  cursor: c;
+}
+
+@media (max-width: 768px) {
+  .forecast-summary {
+    width: calc(50% - 2rem);
```

Comment:
This rule isnt doing anything, also see the point above about this needing to be in a style file that relates to the specific component its affecting rather than a more high level one like this